src/components/app-wrapper.jsx

```diff
@@ -48,6 +50,8 @@ import { BrowserRouter } from 'react-router-dom';
 import { Provider, useSelector } from 'react-redux';
 import messages_en from '../translations/en.json';
 import messages_fr from '../translations/fr.json';
+import exportParameters_en from '../translations/external/export-parameters-en.json';
+import exportParameters_fr from '../translations/external/export-parameters-fr.json';
 import network_modifications_locale_en from '../translations/dynamic/network-modifications-locale-en';
```

Comment:
To remove ?

Reply:
Yes. There was a mixup during the merges and conflict resolution. I removed a few things including this.